README.md

```diff
@@ -4,35 +4,90 @@ NodeJS >=18.19.0
+### Additional Background Information on the Setup
 
-### Notes
-- The `npm run dev:frontend` command runs your application and integrates netlify under the `/admin/index.html` routes. 
+- The `npm run dev:frontend` command runs your application and integrates netlify under the `/admin/index.html` routes.
 - The CMS proxy server enables the use of the local strategy for content management, directly saving files to the local file system.
 
+## Short File Tree Description
+
+```txt
+.
+├── environmentVariables.ts.template // Template file to inject .env variables. Will be filed wit envsubst in the build step. See "Building Details" for more info.
+├── migration-from-bsaber.com // Scripts used to transfer MotW from the old page - can be ignored
+├── README.md
+├── src
+│   ├── app.html // Entry point for the application - contains placeholder for special tags such as <svelte:head> as well
+│   ├── app.scss // Global styles
+│   ├── collections // Actual data for the collection per their model definiton. One folder = One Collection
+│   ├── lib // Components and functions that can be accessed with the $lib path
+│   ├── maps.ts // Central point for enums, that are shared across multiple components
+│   ├── routes // Svelte(kit) routing - these are the pages of our website. See the "Project Routing" section for more details
+│   │   ├── 404.svelte
+│   ├── scss // Styles that are shared across multiple components
+│   │   └── variables.scss // Defines a preset of colors and values to be used uniformly across the project - need a color? Import from here!
+│   └── types.ts // A central place for types - if types are specific for a component (or similar) and not shared across the project, put them in the respective component file
+├── static // These files will be mounted on the root. For example, ./static/example.jpg will later be accessible under https://bsaber.com/example.jpg
+│   ├── admin
+│   │   ├── config.yml // This files defines the config and data model for Netlify CMS - if you want to edit, add or remove a collection some changes will happen here
+│   │   └── index.html // Entry point for the Netlify CMS page for admins
+│   ├── characteristics // Folder containing the images for the diffs on the map cards
+│   ├── favicon.png // This file is the favicon of the website
+│   ├── uploads // Directory, that contains uploaded images from netlify CMS
+├── svelte.config.js // Svelte(kit) configuration - contains the configurations for crawling and SSG as well
+├── update-cover-url-overwrites.sh // See "Utility Scripts" for info
+```
+
+## Project Routing
+
+The routes are determined by file path starting at `src/routes`. For example, (`src/routes/community-hub/+page.svelte`)[src/routes/community-hub/+page.svelte] renders under (https://bsaber.com/community-hub)[https://bsaber.com/community-hub] or in the local development environment under (http://localhost:3000/community-hub)[http://localhost:3000/community-hub].
```

Comment:
This first hyperlink has its brackets flipped, the format for hyperlinks is `[text](url)`.
The second and third, like the ones above, don't need to be hyperlinks explicitly.